src/services/codefixes/fixAddMissingMember.ts

```diff
@@ -315,7 +315,7 @@ namespace ts.codefix {
         sourceFile: SourceFile,
         isJSFile: boolean
     ): void {
-        const importAdder = createImportAdder(sourceFile, context.program, context.preferences, context.host);
+        const importAdder = createImportAdder(sourceFile, context.program, context.autoImportProvider, context.preferences, context.host);

```

Comment:
I don't think this can actually require auto-imports.  Probably the same is true for some of the other code fixes.